drgn/commands/_builtin/crash/_net.py

```diff
@@ -0,0 +1,350 @@
+            "SOCK",
+            "FAMILY:TYPE",
+            "SOURCE-PORT",
+            "DESTINATION-PORT",
+        ]
+    ]
+
+    cnt = 0
+    for fd, fp in for_each_file(task):
+        if not fp:
+            continue
+        try:
+            socket = SOCKET_I(fp.f_path.dentry.d_inode)
+        except ValueError:
+            continue
+
+        sock_family = str(
+            sockfamily_dict.get(int(socket.sk.__sk_common.skc_family))
+        )
+        if (
+            int(socket.sk.__sk_common.skc_family)
+        ) not in sockfamily_dict.keys():
+            sock_family = "AF_"
+
+        sock_family = sock_family[3:]
+        sock_type = str(socktype_dict.get(int(socket.type)))
+
+        if int(socket.type) not in socktype_dict.keys():
+            sock_type = "SOCK_"
+            sock_type = sock_type[5:]
+
+        if sock_family == "INET":
+            skc = socket.sk.__sk_common
+            saddr = get_ipv4(prog, skc.skc_rcv_saddr.address_of_())
+            daddr = get_ipv4(prog, skc.skc_daddr.address_of_())
+            sport = int(skc.skc_num)
+            dport = int(skc.skc_dport)
+            if prog.platform.flags & PlatformFlags.IS_LITTLE_ENDIAN:
+                dport = socketlib.ntohs(dport)
+
+            source_port = f"{saddr}:{sport}"
+            destination_port = f"{daddr}:{dport}"
+
+        elif sock_family == "INET6":
+            skc = socket.sk.__sk_common
+            saddr_6 = get_ipv6(prog, skc.skc_v6_rcv_saddr.address_of_())
+            daddr_6 = get_ipv6(prog, skc.skc_v6_daddr.address_of_())
+            sport_6 = int(skc.skc_num)
+            dport_6 = int(skc.skc_dport)
+            if prog.platform.flags & PlatformFlags.IS_LITTLE_ENDIAN:
+                dport_6 = socketlib.ntohs(dport_6)
+
+            source_port = f"{saddr_6}:{sport_6}"
+            destination_port = f"{daddr_6}:{dport_6}"
+
+        else:
+            source_port = ""
+            destination_port = ""
+
+        cnt = cnt + 1
+        rows.append(
+            [
+                fd,
+                hex(socket.value_())[2:],
+                hex(socket.sk.value_())[2:],
+                sock_family + ":" + sock_type,
+                source_port,
+                destination_port,
+            ]
+        )
+        if print_full_data:
+            print(
+                "%-5s %-20s %-20s %-20s %-20s %-20s"
+                % (
+                    str(cnt),
+                    hex(socket.value_())[2:],
+                    hex(socket.sk.value_())[2:],
+                    sock_family + ":" + sock_type,
+                    source_port,
+                    destination_port,
+                )
+            )
+            print(socket)
+            print(socket.sk)
+
+    if not (print_full_data):
+        print_table(rows)
+
+def print_net_devices(prog: Program) -> None:
+    rows = [["NET_DEVICE", "NAME", "IP ADDRESS(ES)"]]
+    net_namespaces = network.for_each_net(prog)
+
+    for net_namespace in net_namespaces:
```

Comment:
Crash doesn't appear to iterate over all network namespaces. It seems like it defaults to the initial network namespace (`init_net`), so let's do the same.